src/vs/code/node/cli.ts

```diff
@@ -413,6 +413,9 @@ export async function main(argv: string[]): Promise<any> {
 
 		let child: ChildProcess;
 		if (!isMacOSBigSurOrNewer) {
+			if (!verbose && args.status) {
```

Comment:
Can L410 be instead changed to `!verbose || !args.status` ?

Reply:
@deepak1556 I thought about it, but for when we use `open` we do not really want the output of `open` right? We have a different solution in that case to track output.